### ISSUE TITLE
Copy Online Instance - Error ID3242

### ISSUE THREAD

**Issue by AK-RenegadeX** [WaelHamze/xrm-ci-framework#267]
I'm currently attempting to copy an instance from Production to Sandbox and running into the following error.  The same configuration worked in an demo trial tenant.

Federated service at https://xxx.xxx.gov/adfs/services/trust/13/usernamemixed returned error: ID3242: The security token could not be authenticated or authorized.

**Comment by AK-RenegadeX** [comment]
Seems I was pointing to wrong api URL, my error is now: 
Need a non-empty authority.

**Comment by AK-RenegadeX** [comment]
Added the Tenant ID to command, this further defines the error as:
**ERROR REQUESTING Token FROM THE Authentication context**
returns null and then ends in object reference exception.

I've added TLS1.2 statement as suggested by forum with no lock.

**Comment by AK-RenegadeX** [comment]
After using the government service provider, I'm getting this error with both the MSCRM tools and the PowerShell equivalent: **Operation returned an invalid status code 'Forbidden'**

**Comment by WaelHamze** [comment]
@AK-RenegadeX I presume you are using the latest online management API right? Did you try to open a ticket with vendor?

**Comment by AK-RenegadeX** [comment]
Yes, I thought I was using the latest version but not sure how to validate.  I have sent an email to the vendor via the PowerShell Gallery.

Here is a screen shot of my pipeline.


--
Shane

**Comment by WaelHamze** [comment]
@AK-RenegadeX I presume you have managed to fix this?

**Comment by AK-RenegadeX** [comment]
Not yet but I believe it has something to do with the configuration of the Dynamics instances as they are in GCC and could be limited by the additional security restrictions.

**Comment by WaelHamze** [comment]
@AK-RenegadeX I presume the account you are running this under is a Dynamics 365 Service Admin and has access to both of the instances?

Also is MFA enabled for this account?

**Comment by AK-RenegadeX** [comment]
Unfortunately the company that owns the instances doesn’t grant this privilege, I am using a System Administrator account to do the copy.

**Comment by WaelHamze** [comment]
@AK-RenegadeX can you able to perform a copy from the UI using the same acccount? https://docs.microsoft.com/en-us/power-platform/admin/copy-environment
